src/js/components/order.js

```diff
@@ -10,6 +11,53 @@ import PanelGroup from 'react-bootstrap/lib/PanelGroup';
 import Row from 'react-bootstrap/lib/Row';
 import Table from 'react-bootstrap/lib/Table';
 
+class AmountField extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: this.props.value != null ? parseInt(this.props.value) : 0
```

Comment:
replace ```null``` with ```undefined```
If someone doesn't pass props.value it will return ```undefined``` that isn't equal to ```null```.